### ISSUE TITLE
Model Builder error window displayed outside of border for long messages 

### ISSUE THREAD

**Issue by zewditu** [dotnet/machinelearning-modelbuilder#2358]
**System Information (please complete the following information):**
 - Model Builder Version (available in Manage Extensions dialog): 
 - Visual Studio Version

**Describe the bug**
- On which step of the process did you run into an issue:
- Clear description of the problem:

**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/36615490/200704656-fb9065b0-1a76-4fbc-96cd-9df43431574a.png)

**Additional context**

**Comment by zewditu** [comment]
Add the error message into ScrollViewer however two ScrollViewer in a single window ?

**Comment by beccamc** [comment]
Yes let's add a scroll for the message. Make sure that it only goes to scroll when it's larger than a certain size. It's weird but its fine.
